Charts/Classes/Renderers/CandleStickChartRenderer.swift

```diff
@@ -43,7 +43,7 @@ public class CandleStickChartRenderer: LineScatterCandleRadarChartRenderer
         }
     }
     
-    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 4)
+    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 2)
```

Comment:
haha ITS MY MISTAKE :P, I removed the count and when I added it, I added it back as 2. IT SHOULD BE 4. I'll update it in the next commit